src/Cronos/CronExpression.cs

```diff
@@ -128,31 +170,30 @@ internal CronExpression(
 
         ///<summary>
         /// Constructs a new <see cref="CronExpression"/> based on the specified
-        /// cron expression. It's supported expressions consisting of 5 fields:
+        /// cron expression. Its supported expressions consist of 5 fields:
         /// minute, hour, day of month, month, day of week. 
-        /// If you want to parse non-standard cron expressions use <see cref="Parse(string, CronFormat)"/> with specified CronFields argument.
         /// See more: <a href="https://github.com/HangfireIO/Cronos">https://github.com/HangfireIO/Cronos</a>
         /// </summary>
-        public static CronExpression Parse(string expression)
+        public static CronExpression Parse(string expression, Random? rng = null)
```

Comment:
I would propose adding a new overload for the `Parse` method, instead of introducing an optional parameter, since the latter is considered to be an unnecessary breaking change.